### ISSUE TITLE
Make Ingress resource compatible with kubernetes versions lower than 1.19

### ISSUE THREAD

**Issue by joseacl** [provectus/kafka-ui#2317]
**Describe the bug**
The helm template file for ingress is checking that **APIVersions** contains **networking.k8s.io/v1**, but this capability is available in Kubernetes since the version v1.8 **but for NetworkPolicy resource and not for Ingress**. https://kubernetes.io/docs/reference/using-api/deprecation-guide/#networkpolicy-v116

So if you applied helm template with **--kube-version 1.18** for example, the ingress is going to be rendered using **apiVersion: networking.k8s.io/v1** which is not supported by this version of Kubernetes https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122

**Set up**
Using helm version Version:"v3.8.0" with has --kube-version flags available.

Clone the repository and go to charts/kafka-ui

**Steps to Reproduce**

1. Execute the following command
`helm template --kube-version 1.18 --set ingress.enabled=true --output-dir ./manifests .`

**Expected behavior**
Ingress resource rendered should be compatible with kubernetes version specified, 1.18. Something like this

```
apiVersion: networking.k8s.io/v1beta1
kind: Ingress
metadata:
  name: release-name-kafka-ui
  labels:
    helm.sh/chart: kafka-ui-0.0.4
    app.kubernetes.io/name: kafka-ui
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/version: "latest"
    app.kubernetes.io/managed-by: Helm
spec:
  rules:
    - http:
        paths:
          - backend:
              serviceName: release-name-kafka-ui
              servicePort: 80
```

But with the current template, the ingress is incompatible with versions lower than 1.19

```
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: release-name-kafka-ui
  labels:
    helm.sh/chart: kafka-ui-0.0.4
    app.kubernetes.io/name: kafka-ui
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/version: "latest"
    app.kubernetes.io/managed-by: Helm
spec:
  rules:
    - http:
        paths:
          - backend:
              service:
                name: release-name-kafka-ui
                port:
                  number: 80
            pathType: Prefix

```

**Comment by joseacl** [comment]
Hi @Haarolean @germanosin.

I've opened that issue because we're rolling out this tool in our clusters and due to the fact that we have clusters with different Kubernetes versions, we've realized the error described in the issue.

Currently, we're overriding the ingress template, so I've decided to create a PR https://github.com/provectus/kafka-ui/pull/2318 because it can happen to other users.
Looking forward to hearing from you, 

Thanks a lot!
